aten/src/THCUNN/generic/VolumetricReplicationPadding.cu

```diff
@@ -11,7 +11,7 @@ static inline void THNN_(VolumetricReplicationPadding_shapeCheck)(
                          int pfront, int pback) {
   THArgCheck(THCTensor_canUse32BitIndexMath(state, input), 2,
              "input tensor must fit into 32-bit index math");
-  int numInputDims = THCTensor_(nDimension)(state, input);
+  int numInputDims = THCTensor_(_nDimension)(state, input);
```

Comment:
please revert aten files changes, they shouldn't be needed

Reply:
yes, need to rebase and do some more clean-ups. thx